tldw_Server_API/app/core/DB_Management/media_db/repositories/media_files_repository.py

```
@@ -0,0 +1,207 @@
+            "uuid": new_uuid,
+            "created_at": now,
+            "last_modified": now,
+            "version": 1,
+            "client_id": db.client_id,
+            "deleted": 0,
+            "prev_version": None,
+            "merge_parent_uuid": None,
+        }
+        placeholders = ", ".join([f":{key}" for key in data])
+        columns = ", ".join(data.keys())
+        sql = f"INSERT INTO MediaFiles ({columns}) VALUES ({placeholders})"  # nosec B608
```

Ruff (0.15.6)

[error] Possible SQL injection vector through string-based query construction

(S608)